app/models/service.rb

```diff
@@ -52,17 +42,23 @@ class Service < ActiveRecord::Base
               state_province: a.state_province,
               postal_code: a.postal_code,
               country: a.country,
-              address_1: a.address_1
+              address_1: a.address_1,
+              latitude: a.latitude.to_f || nil,
+              longitude: a.longitude.to_f || nil,
```

Comment:
added to match what the UI expects in https://github.com/ShelterTechSF/askdarcel-web/pull/894/